command/command_test.go

```diff
@@ -89,13 +90,24 @@ func testVaultServerAllBackends(tb testing.TB) (*api.Client, func()) {
 	return client, closer
 }
 
+// testVaultServerAutoUnseal creates a test vault cluster and sets it up with auto unseal
+// the function returns a client, the recovery keys, and a closer function
+func testVaultServerAutoUnseal(tb testing.TB) (*api.Client, []string, func()) {
+	testSeal := seal.NewTestSeal(nil)
+	autoSeal, err := vault.NewAutoSeal(testSeal)
```

Comment:
I'm not sure if there's a better way to set up auto-unseal for testing

Reply:
Depends on what you're trying to do.  I think you made the right choice here.  If we were actually exercising seal related more directly, I might point you at something that doesn't use a NewTestSeal, which is kind of a fake.